Python/chapter01/1.3 - URLify/lomeli_1.3.py

```diff
@@ -0,0 +1,14 @@
+# Write a method to replace all spaces in a string with '%20'
+# "Mr John Smith     ", 13 => "Mr%20John%20Smith"
+
+def Urlify(string, size):
+    temp = []
```

Comment:
nit: it might be better to give this a more [descriptive name](https://medium.com/coding-skills/clean-code-101-meaningful-names-and-functions-bf450456d90c) than `temp`. something like `urlified` could work.